modules/migration-configuring-mcg.adoc

```diff
@@ -42,7 +42,10 @@ ifdef::installing-3-4,installing-mtc[]
 You use these credentials to add MCG as a replication repository.
 endif::[]
 ifdef::installing-oadp-mcg[]
-. Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` by running the link:https://docs.redhat.com/en/documentation//red_hat_openshift_data_foundation/4.15/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the `NooBaa` custom resource.
+. Create a MCG bucket. For details, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.17/html-single/managing_hybrid_and_multicloud_resources/index[Managing hybrid and multicloud resources].
+
+. Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and bucket name by running the link:https://docs.redhat.com/en/documentation//red_hat_openshift_data_foundation/4.15/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the bucket resource.
```

Comment:
`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` should be functioning as adjectives, e.g. "`AWS_ACCESS_KEY_ID` value" or something.

Reply:
Is linking to the command important here?